source/ada/lsp-ada_documents.adb

```diff
@@ -1139,22 +1159,55 @@ package body LSP.Ada_Documents is
 
             Insert_Text := Insert_Text & " (";
 
+            --  Compute number of params to know if named notation should be
+            --  used.
+
+            for Param of Params loop
+               for Id of Param.F_Ids loop
+                  Nb_Params := Nb_Params + 1;
```

Comment:
how about removing the inner loop and do

    Nb_Params := Nb_Params + Param.Length;

?

Reply:
Done, thanks for noticing!